### ISSUE TITLE
Code and Checkpoint

### ISSUE THREAD

**Issue by sutwangyan** [zhoubenjia/GFSLT-VLP#4]
Hello, thank you for your excellent work.
1.In the train_slt.py file.
`
if 'text_decoder' in state_dict:
     for k, v in state_dict['text_decoder'].items():
          if 'decoder' in k:
              k = 'mbart.model.decoder.'+'.'.join(k.split('.')[2:])
              new_state_dict[k] = v
`
Should k.split('.')[2:] be modified to k.split('.')[1:]?

2.Can you provide log and checkpoint files? During my replication process, the DEV Loss stopped decreasing after reaching 3.79

**Comment by zhoubenjia** [comment]
Hi, thank you for your attention. As for the first question, I think it should not be like this. If we change it to:
k = 'mbart.model.decoder.'+'.'.join(k.split('.')[1:])
it seems that we cannot initialize the text decoder.
Regarding the second issue,  the DEV/loss seems a bit high, it may indicate overfitting of the model on the training data. Please reference our [log files](https://github.com/zhoubenjia/GFSLT-VLP/tree/main/log). Special attention should be paid to setting CUDA_VISIBLE_DEVICES if you are training with certain GPUs on a single node.
